mongomock/aggregate.py

```diff
@@ -157,7 +160,11 @@ def parse(self, expression):
                 return self._handle_conditional_operator(k, v)
             if k in set_operators:
                 return self._handle_set_operator(k, v)
-            if k in boolean_operators + string_operators + \
+            if k in string_operators:
+                return self._handle_string_operator(k, v)
+            if k in misc_operators:
+                return self._handle_misc_operator(k, v)
+            if k in boolean_operators + \
```

Comment:
Please update the Missing_features.rst file to mention that those features are not missing anymore.

Reply:
np